articles/quickstart/webapp/aspnet-core-2/_includes/_login.md

```diff
@@ -307,7 +321,7 @@ The OIDC middleware in ASP.NET Core automatically decodes the ID Token returned
 
 The seed project contains a controller action and view that display the claims associated with a user. Once a user has logged in, you can go to `/Account/Claims` to see these claims.
 
-In order to be able to retrieve the Access Token to authenticate the user in calls to your API, update the Startup.cs file so that, when calling `AddOpenIdConnect`, the `SaveTokens` property is set to `true`.
+You may want to Access Tokens received from Auth0. For example, you can use the Access Token to authenticate the user in calls to your API. To achieve this, when calling `AddOpenIdConnect`, set the `SaveTokens` property to `true`. This saves the tokens to `AuthenticationProperties`:
```

Comment:
Sounds like there are a few words missing in : `You may want to Access Tokens`.

Reply:
Yes looks like it, I've modified the text here to:

> You may want to store the tokens received from Auth0. For example, you can use the Access Token later to authenticate the user in calls to your API.

What do you think?

Reply:
Yes, great!